config/label-management.md

```diff
@@ -0,0 +1,126 @@
+# Managing cross-organization labels in existing repositories #
+
+This is a step-by-step guide/script for managing organization-wide github
+issue labels across all repositories.
+
+## Prerequisites ##
+
+- User has access to the github organization's repositories
+- User can generate a GitHub [Personal access token](https://github.com/settings/tokens)
+
+## Initial Setup ##
+
+1. Clone the repository locally
+
+```console
+git clone https://github.com/alta3/github-label-management.git
+cd github-label-management
+```
+
+1. Activate the virtual environment and install the requirements
+
+```console
+python3.8 -m venv venv
+source venv/bin/activate
+python -m pip install --upgrade pip wheel
+python -m pip install -r requirements.txt
+```
+
+1. Create a local configuration file by running `touch config`
+1. Open `config` in your favorite editor for editing
+1. Create a GitHub [Personal access token](https://github.com/settings/tokens)
```

Comment:
This ordered list is not rendering correctly.  I think some of the blank lines need to be removed, and/or the console sections need to be indented by four spaces.

You probably need four leading spaces before the unordered list items, assuming they should be under the "Create a GitHub personal access token..." item.

Reply:
Resolved in https://github.com/cisagov/development-guide/pull/41/commits/105129a08608094e93141604071ab63eabb2dae1

Reply:
I think this is [partially resolved, but not completely](https://github.com/cisagov/development-guide/blob/label-mgmt/config/label-management.md).  The numbering of the ordered list items restarts at one a few times: 1, 1, 1, 2, 3, 1 instead of 1, 2, 3, 4, 5, 6.

Reply:
Good catch! https://github.com/cisagov/development-guide/pull/41/commits/c6b0764dfd25147e15cc89e07140e05436d65d53

Reply:
Thanks for adding commit c6b0764 and fixing the numbering!